invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -46,6 +46,9 @@
     margin-left: 5px;
   }
 
+  #community-upload-button {
+    padding: 0px;
```

Comment:
what is the problem you are trying to solve by this custom css? can we avoid it ?

Reply:
https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides#L60 that rule was adding some padding. The problem is that class not my current implementation but I didn't want to dig into what is the impact on UI by changing that rule...

Reply:
Discussed IRL to document and solve the source of the problem i.e avoid custom styling and implement a maintable way of adding styles. I created a placeholder issue for that https://github.com/inveniosoftware/invenio-app-rdm/issues/1265